LanternExtractor/EQ/Wld/Helpers/CharacterFixer.cs

```diff
@@ -389,6 +390,29 @@ private void FixHalasFemale()
             RenameBone(skeleton, 15, "head_point");
         }
 
+        /// <summary>
+        /// Early beta beetles have SPI named bones
+        /// </summary>
+        private void FixBetaBeetle()
+        {
+            var skeleton = _wld.GetFragmentByName<SkeletonHierarchy>("BET_HS_DEF");
+
+            if (skeleton == null)
```

Comment:
This is a good pattern we can apply to the others fixes which iterate through and continue on mismatched names.